### DISCUSSION TITLE
Way to seed data into mongo?

### DISCUSSION THREAD

**Discussion by edeandrea** [quarkusio/quarkus#24638] [Q&A] [answered]
Is there a way to seed initial data using the mongo with panache extension? Something similar to the `import.sql` used by the hibernate extensions?

[a comment by an automated account is not shown]

**Comment by loicmathieu** [comment, marked as answer]
Yes, via the Liquibase MongoDB extension, see https://quarkus.io/guides/liquibase-mongodb

**Reply by edeandrea** [reply]
Thanks I'll give that a look. One thing I noticed - in the guide it says

```
add your changeLog to the src/main/resources/db/changeLog.xml file as you usually do with Liquibase
```

yet in the quickstart, it shows [`changelog.xml` in the `src/main/resources/liquibase` directory](https://github.com/quarkusio/quarkus-quickstarts/blob/main/liquibase-mongodb-quickstart/src/main/resources/liquibase/changelog.xml). Notice the name of the file and its location is different than what the guide says.

Which is correct?

**Reply by loicmathieu** [reply]
Both !
The quickstart configure the default liqubase changelog location, see https://github.com/quarkusio/quarkus-quickstarts/blob/main/liquibase-mongodb-quickstart/src/main/resources/application.properties

**Reply by edeandrea** [reply]
Aha thats the missing link!

**Reply by edeandrea** [reply]
That works - thanks!

One more question - Is there any particular reason that you know of as to why the dev services for mongo uses version `4.0.10` rather than a more recent version (5.x)?

**Reply by loicmathieu** [reply]
No idea, I didn't work on the dev services for MongoDB
